src/H5Dlayout.c

```diff
@@ -42,11 +42,13 @@
 
 /* Format version bounds for layout */
 const unsigned H5O_layout_ver_bounds[] = {
-    H5O_LAYOUT_VERSION_1,                      /* H5F_LIBVER_EARLIEST */
-    H5O_LAYOUT_VERSION_3, /* H5F_LIBVER_V18 */ /* H5O_LAYOUT_VERSION_DEFAULT */
-    H5O_LAYOUT_VERSION_4,                      /* H5F_LIBVER_V110 */
-    H5O_LAYOUT_VERSION_4,                      /* H5F_LIBVER_V112 */
-    H5O_LAYOUT_VERSION_LATEST                  /* H5F_LIBVER_LATEST */
```

Comment:
This change looks a little suspect, though in the 1.14 branch `H5O_LAYOUT_VERSION_LATEST == H5O_LAYOUT_VERSION_4` so I'm not sure that it's a problem for the array to be setup like this.

Reply:
Fixed